### PR TITLE
post method updated

### PR DESCRIPTION
**Pull request by dekim2324**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -28,8 +28,28 @@ const getAllPhotos = (id, callback) => Photos.find({ product_id: id }, {}, { sor
...
+    image_url: seed.image_url,
+  });
+
+  photos.save((err, pictures) => {
+    if(err) return console.error(err);
+    console.log('pictures saved successfully!!', pictures)
+  })
+
+}
+
```
Comment (mflinn92) [inline database/index.js]: Was the previous seeding script broken?

File: server/server.js [hunk 2]
```diff
@@ -24,12 +26,22 @@ app.use(cors());
...
     res.json(result);
   });
 });
 
+app.post('/api/photos', (req, res) => {
+  for(let i = 0; i < 8; i++) {
+    let oneSeed = seed.seeds[i];
+    db.savePhotos(oneSeed);
+  }
+
```
Comment (mflinn92) [inline server/server.js]: Should the post request be able to save new photos that are attached to the response body?

Comment (dekim2324) [reply server/server.js]: yes you're right actually, good catch! will fix and update

Comment (dekim2324) [reply server/server.js]: @mflinn92

**Pull request closed without merging**